components/dashboard/src/index.tsx

```diff
@@ -64,19 +63,17 @@ const bootApp = () => {
                             <AdminContextProvider>
                                 <PaymentContextProvider>
                                     <LicenseContextProvider>
-                                        <TeamsContextProvider>
-                                            <ProjectContextProvider>
-                                                <ThemeContextProvider>
-                                                    <StartWorkspaceModalContextProvider>
-                                                        <BrowserRouter>
-                                                            <FeatureFlagContextProvider>
-                                                                <App />
-                                                            </FeatureFlagContextProvider>
-                                                        </BrowserRouter>
-                                                    </StartWorkspaceModalContextProvider>
-                                                </ThemeContextProvider>
-                                            </ProjectContextProvider>
-                                        </TeamsContextProvider>
+                                        <ProjectContextProvider>
+                                            <ThemeContextProvider>
+                                                <StartWorkspaceModalContextProvider>
+                                                    <BrowserRouter>
+                                                        <FeatureFlagContextProvider>
+                                                            <App />
+                                                        </FeatureFlagContextProvider>
+                                                    </BrowserRouter>
+                                                </StartWorkspaceModalContextProvider>
+                                            </ThemeContextProvider>
+                                        </ProjectContextProvider>
```

Comment:
Is there any equivalent syntax which would allow us to list these and it would compose them into this pyramid?

Reply:
You don't like pyramids? 😅

Reply:
Could be if we got creative. I'm happy this removes one of the context's though, so at least the pyramid is shrinking 😄 .